article/app/controllers/LiveBlogController.scala

```diff
@@ -39,6 +39,8 @@ class LiveBlogController(
   // we support liveblogs and also articles, so that minutes work
   private def isSupported(c: ApiContent) = c.isLiveBlog || c.isArticle
 
+  private[this] val logger = DotcomponentsLogger()
```

Comment:
I think this could also be added to (and imported from)  the DotcomponentsLogger object